x-pack/plugins/saved_objects_tagging/public/management/components/table.tsx

```diff
@@ -187,6 +191,10 @@ export const TagTable: FC<TagTableProps> = ({
       items={tags}
       pagination={tablePagination}
       sorting={sorting}
+      tableCaption={i18n.translate('xpack.savedObjectsTagging.management.table.columns.caption', {
+        defaultMessage: 'Tags',
+      })}
```

Comment:
EUI provides a super generic table caption but providing our own is a good way to add meaningful structure to the page. It doesn't have to be anything fancy either, just describing the table contents is great.